Transcriptions

ILobzhanidze opened the pull request:
Dear Andrew,

I have cleaned my local repository. There are only .xml files. 

Sincerely yours,
Irina

ILobzhanidze committed:
Test Transcription

andrew-morrison committed:
Update transcriptions branch with changes on master

ILobzhanidze committed:
Changes in the transcription of hand

ILobzhanidze committed:
New Changes

ILobzhanidze committed:
Changes on MS Georg d 2

ILobzhanidze committed:
Recommiting

andrew-morrison committed:
Deleting HTML file

andrew-morrison committed:
Deleting XSL file

andrew-morrison commented:
You have made changes to the font in the preview.css file. Is this because Georgian characters weren't being displayed properly, or is it just a personal preference?

ILobzhanidze commented:
The main problem, which I faced was displaying of Georgian characters: a) together with combining double tilde: character + &#x0360; (from Unicode chart 0360) and b) non syllabic u: უ̂ (from Unicode chart 10e3 + 0302)

andrew-morrison commented:
And did changing the fonts in preview.css fix those issues?

If so, we may have to investigate whether any royalty-free fonts which can be used on the georgian.bodleian.ox.ac.uk web site provide the same character support. We can specify the same fonts you've used, but they will only work for visitors which have them installed on their computers.

ILobzhanidze commented:
I have seen the fonts indicated in preview.css, especially: Baskerville, Palatino Linotype, Palatino, Century Schoolbook L, Times New Roman, serif. After the cloning of repository I have not changed these fonts in preview.css. I worked only in Oxygen in 'collections' folder, where I turned my preferences to DejavuSans (I mean in Oxygen), because of the problems mentioned above. In case of Georgian the best font family is DejaVu Sans Font Family.

ILobzhanidze committed:
Updates on MS Georg d 2

MS Wardr c 4 and MS Wardr e 14 are not finished yet.

ILobzhanidze committed:
Last update on the Typicon of Holy Cross

I have aligned texts with photos and provided linking to photo material available on Digital Bodleian.

ILobzhanidze committed:
Updates on manuscripts

ILobzhanidze committed:
Some additions

ILobzhanidze committed:
Updates on files

ILobzhanidze committed:
Html removed

ILobzhanidze committed:
Text updates

The texts of MS. Georg. d. 2; MS. Wardr. c. 1 (1); MS. Wardr. c. 1 (2); MS. Wardr. c. 1(3); MS. Wardr. c. 1 (39), MS. Wardr. e. 14; MS. Wardr. f. 2; MS. Wardr. g. 2 were updated

ILobzhanidze committed:
update on MS Georg d. 2 and MS Wardr c. 4

ILobzhanidze committed:
Update MS_Georg_d_2.xml

ILobzhanidze committed:
Addition of MS_Wardr_d_1

ILobzhanidze committed:
Addition of MS_Wardr_e_32

ILobzhanidze committed:
Update README.md

ILobzhanidze committed:
Update README.md

ILobzhanidze committed:
Update

ILobzhanidze committed:
Update MS_Wardr_e_32.xml

ILobzhanidze committed:
Addition of new transcriptions

ILobzhanidze committed:
Ms_Wardr_e_33

ILobzhanidze committed:
Update

ILobzhanidze committed:
Update
